src/NearCache.md

```diff
@@ -107,6 +118,11 @@ Following are the descriptions of all configuration elements:
 - `local-update-policy`: Specifies the update policy of the local Near Cache. Is just available on JCache clients. Available values are as follows:
    - `INVALIDATE`: Does not update the local Near Cache. Will invalidate the local Near Cache eventually (default value).
    - `CACHE`: Updates the local Near Cache immediately after the put operation completes.
+- `preloader`: Specifies if the Near Cache should store and pre-load its keys for a faster re-population after a Hazelcast client restart. Is just available on IMap and JCache clients. It has the following attributes:
+  - `enabled`:
+  - `file-name`:
+  - `store-initial-delay-seconds`:
+  - `store-interval-seconds`:
 
```

Comment:
Specifies if the Near Cache should store and preload its keys for a faster re-population after a Hazelcast client restart. It is available only for IMap and JCache clients and has the following attributes:

??? the attributes need explanations, or claimed to be self-explanatory ???